proto/internal/temporal/server/api/token/v1/message.proto

```diff
@@ -85,3 +86,13 @@ message NexusOperationCompletion {
     // Allows completing a started operation after a workflow has been reset.
     string request_id = 5;
 }
+
+// A completion token for a Nexus operation started from a CHASM component.
+message NexusOperationChasmCompletion {
```

Comment:
Hmm... can we merge this into the existing `NexusOperationCompletion` and deprecate the original fields?

You can just add a `component_ref` field and it would work well.

Not critical but it would simplify the decoding later.